### PR TITLE
Make the Certificate and Roles tables design as the rejection reason table

### PR DESCRIPTION
**Pull request by ilja903**
https://unctad.atlassian.net/browse/EV2CORE-66

### CODE REVIEW

**Comment by kamsi** [comment]
@ilja903 fix the conflicts, please.

**Comment by ilja903** [comment]
resoved

File: view/statistics-flow-certificates.js [hunk 1]
```diff
@@ -140,10 +140,10 @@ exports['statistics-main'] = function () {
 			p({ class: 'submit' }, input({ type: 'submit' }))),
 		section({ class: 'pad-if-pagination' }, pagination),
 		br(),
-		section({ class: "section-primary" },
 			data.map(function (result) {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```
Comment (kamsi) [inline view/statistics-flow-certificates.js]: I'm not sure that those classes are needed. However, what is needed is `responsive-container` class (See rejection reasons table for reference).

File: view/statistics-flow-roles.js [hunk 2]
```diff
@@ -173,10 +173,10 @@ exports['statistics-main'] = function () {
 
 		section({ class: 'pad-if-pagination' }, pagination),
 		br(),
-		section({ class: "section-primary statistics-table-scrollable" },
 			data.map(function (result) {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```
Comment (kamsi) [inline view/statistics-flow-roles.js]: responsive container may not work here... still that should not happen:

layout_problem

**Comment by ilja903** [comment]
Great code review, width class was totally useless, I removed it. Also fixed the mobile version. However put the comment to Jira next time, Vianney does not read github.

**Pull request merged**